src/ValidationError.js

```diff
@@ -2,7 +2,13 @@
 let strReg = /\$\{\s*(\w+)\s*\}/g;
 
 let replace = str =>
-  params => str.replace(strReg, (_, key) => params[key] || '')
+  params => str.replace(strReg, (_, key) => {
+    const val = params[key]
+    if (typeof val === 'number') {
+      return val
+    }
+    return val || ''
```

Comment:
Not sure. Was just trying to keep the same intent that the original coder had (defaulting to `'`), I just added and exception `0` issue. I can certainly go that route if you would like?